### PR TITLE
DEVCVL-201 - Home Page Implemented

### PR DESCRIPTION
**Pull request by kyawthetkt**
Please view this video ->
https://drive.google.com/file/d/1iMi9CbOnGE9xCule-z4WGdPKQCdQFmgZ/view.

Finalize Home page with tailwind css installation.

### CODE REVIEW

**Comment by kyawthetkt** [comment]
Please view this video ->
https://drive.google.com/file/d/1iMi9CbOnGE9xCule-z4WGdPKQCdQFmgZ/view.

**Review by shakil-adv — APPROVED** [review]
Well done on the responsiveness!

**Review by pracheebapate — APPROVED** [review]

**Review by jibrail-idris — CHANGES_REQUESTED** [review]

File: src/pages/homepage/Home.js [hunk 1]
```diff
@@ -1,26 +1,102 @@
...
+    const data = response?.data ? response?.data : [];
 
     return (
-        <div>
-            {/* <Card {...sampleCardProp} /> */}
-            <Collection
-                data={[sampleCardProp, sampleCardProp, sampleCardProp]}
-            />
-        </div>
+        <>
```
[the remark this answers is not in the document]
Comment (kyawthetkt) [reply src/pages/homepage/Home.js:46]: @jibrail-idris No, it is not needed.Will remove it!

[the remark this answers is not in the document]
Comment (kyawthetkt) [reply src/pages/homepage/Home.js]: @jibrail-idris Fixed.

**Review by jibrail-idris — APPROVED** [review]

**Pull request merged**